portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -56,20 +56,48 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     response.sendRedirect("/walkthrough/");
   }
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("application/json;");
+  private int getCommentLimit(String commentLimitString) {
```

Comment:
hypothetically, imagine there was a malicious client who wanted to crash your server somehow. Let's say there were 1 billion comments and they passed in >1 billion as a number.

a good best practice is to enforce some kind of limit on the server side, even if a malicious client requests billions, to have some kind of max.